src/Token.php

```diff
@@ -26,6 +26,11 @@
     public function headers(): DataSet;
 
     /**
+     * Returns the token claims
+     */
+    public function claims(): DataSet;
```

Comment:
@lcobucci Even if you are going to implement JWE functionality, i don't understand how are you think to get claims from JWT today and in future?